src/main/kotlin/app/revanced/patches/youtube/layout/hide/getpremium/bytecode/patch/HideGetPremiumVideoAdvertisementPatch.kt

```diff
@@ -0,0 +1,63 @@
+package app.revanced.patches.youtube.layout.hide.getpremium.bytecode.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.extensions.instruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.youtube.layout.hide.getpremium.annotations.HideGetPremiumVideoAdvertisementCompatibility
+import app.revanced.patches.youtube.layout.hide.getpremium.bytecode.fingerprints.GetPremiumVideoAdvertisementViewFingerprint
+import app.revanced.patches.youtube.layout.hide.getpremium.resource.patch.HideGetPremiumVideoAdvertisementResourcePatch
+import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
+import org.jf.dexlib2.iface.instruction.TwoRegisterInstruction
+
+@Patch
+@DependsOn([IntegrationsPatch::class, HideGetPremiumVideoAdvertisementResourcePatch::class])
+@Name("hide-get-premium")
+@Description("Hides advertisements for youtube premium in the video player.")
+@HideGetPremiumVideoAdvertisementCompatibility
+@Version("0.0.1")
+class HideGetPremiumVideoAdvertisementPatch : BytecodePatch(
```

Comment:
In favour of being consistent with the @Name annotation:
```suggestion
class HideGetPremiumAdvertisementPatch : BytecodePatch(
```